Libraries/Components/ScrollView/RecyclerViewBackedScrollView.android.js

```diff
@@ -113,6 +113,18 @@ var RecyclerViewBackedScrollView = React.createClass({
       );
     });
 
+    const refreshControl = this.props.refreshControl;
+    if (refreshControl) {
+      // Wrap the NativeAndroidRecyclerView with a AndroidSwipeRefreshLayout.
+      return React.cloneElement(
+        refreshControl,
+        {style: props.style},
```

Comment:
Can you remove props.style above in the`props` definition and instead pass in `this.props.style` (and add flex: 1 if it's necessary).

And set `style` in the `props` definition above to just `{flex: 1}`.

Also rename `props` to `recyclerProps`.